compiler/rustc_metadata/src/rmeta/encoder.rs

```diff
@@ -1225,7 +1225,7 @@ impl EncodeContext<'a, 'tcx> {
             hir::ItemKind::Mod(ref m) => {
                 return self.encode_info_for_mod(item.hir_id, m, &item.attrs);
             }
-            hir::ItemKind::ForeignMod(_) => EntryKind::ForeignMod,
+            hir::ItemKind::ForeignMod{..} => EntryKind::ForeignMod,
```

Comment:
```suggestion
            hir::ItemKind::ForeignMod { .. } => EntryKind::ForeignMod,
```
auto formatting inside of macros